tests/e2e/pageObjects/settings-page.ts

```diff
@@ -25,6 +25,18 @@ export class SettingsPage extends BasePage {
     commandsInPipelineInput = Selector('[data-testid=pipeline-bunch-input]');
     pipelineLink = Selector('[data-testid=pipelining-link]');
 
+    //Date and Time Format
+    selectFormatDropdown = Selector('[data-test-subj=select-datetime]');
+    selectTimezoneDropdown = Selector('[data-test-subj=select-timezone]');
+    getDateTimeOption = (option: string): Selector =>
+        Selector(`[data-test-subj^=date-option-${option}]`);
+    getZoneOption = (option: string): Selector =>
```

Comment:
can you move these 2 functions a bit below, below selectors? with getDateTimeOption